_inc/client/components/jumpstart/index.jsx

```diff
@@ -53,21 +71,19 @@ class JumpStart extends React.Component {
 		return (
 			<div className="jp-jumpstart">
 				<p>
-					{ __( "We're now collecting stats, securing your site, and speeding up your images. Pretty soon you'll be able to see everything going on with your site right through Jetpack! Welcome aboard." ) }
+					{ __(
+						"We're now collecting stats, securing your site, and speeding up your images. Pretty soon you'll be able to see everything going on with your site right through Jetpack! Welcome aboard."
+					) }
 				</p>
 
-				<p>
-					{ this.activateButton() }
-				</p>
+				<p>{ this.activateButton() }</p>
```

Comment:
maybe this wrapping `<p>` is not needed

Reply:
Removed in 83ab35073544903f1606959289368cabf2e8b94c